plugins/module_utils/network/ios/facts/bgp_address_family/bgp_address_family.py

```diff
@@ -33,6 +35,16 @@ class Bgp_AddressFamilyFacts(object):
     def __init__(self, module, subspec="config", options="options"):
         self._module = module
         self.argument_spec = Bgp_AddressFamilyArgs.argument_spec
+        spec = deepcopy(self.argument_spec)
```

Comment:
these lines are not related to next_hop_self revert, had removed these lines to increase TC coverage